openstudio_measure_tester.gemspec

```diff
@@ -41,7 +43,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'minitest-reporters', '1.2.0'
   spec.add_dependency 'rake', '12.3.1'
   spec.add_dependency 'rubocop-checkstyle_formatter', '0.4'
-  spec.add_dependency 'simplecov', '0.18.1'
+  spec.add_dependency 'simplecov', '0.18.2'
```

Comment:
This is the main change of this PR

Reply:
`~> 0.18.0` perhaps actually. Looking at release notes, it seems they break stuff in minor releases since this is alpha (pre major>=1), but patch releases are stable. so any 0.18.x would do
(cf https://github.com/colszowka/simplecov/releases)